_includes/codepens/readonly-demo/index.html

```diff
@@ -0,0 +1,26 @@
+<script type="text/javascript">
+  function getEditorStatus (editorId) {
+    return tinymce.get(editorId).mode.get();
+  };
+  function setEditorStatus (editorId, currentStatus) {
```

Comment:
This might be better named `toggleEditorStatus` as that's what it's really doing (I must have been half awake when I reviewed last time sorry).

Reply:
fixed by https://github.com/tinymce/tinymce-docs/pull/1601/commits/87dc4ff3bc8862d32151713a7f64b1a116ccfcae